walkthrough/runtime.md

```diff
@@ -0,0 +1,5 @@
+### Install a JDK
+
+- Explain why to install JDK
```

Comment:
need to enrich the content of this markdown file. cc @nickzhums

Reply:
what's the main for this content again?

Reply:
For this step, the main purpose is o 
- provide a way to install new jdk
- let users know that at least one Java runtime is required.

So I'm not sure what content to put it here. Currently I prefer to provide a markdown with some descriptions just like python's?

Reply:
sounds good. let me think about it